.api-report/mafs.api.md

```diff
@@ -135,15 +135,15 @@ export interface ParametricProps extends Stroked {
     // (undocumented)
     color?: string;
     // (undocumented)
-    samples?: number;
+    minimumSamplingDepth?: number;

```

Comment:
Previously, `samples` was the number of samples the renderer would take of the xy function from t<sub>i</sub> to t<sub>f</sub>. Now, this `depth` parameter controls the minimum bound for how many times the renderer will subdivide the function to minimize the jaggedness of the curve.